README.md

```diff
@@ -498,13 +493,15 @@ Some details on how this works:
   liquidation.
 - 🄳
   [**Liquidation overhang**](https://equityzen.com/blog/startup-valuations-and-liquidation-preference-overhang/)
-  can occur when the value of a company just doesn’t reach the number of dollars investors
-  put into it. Because of liquidation preference, those holding preferred stock (investors)
-  will have to be paid before those holding common stock (employees).
-  If investors invested a hundred million dollars in your company, your equity as an
-  employee won’t be worth anything if the company is sold when it’s in liquidation overhang
-  and the sale doesn’t
-  [exceed that amount](https://avc.com/2010/10/employee-equity-the-liquidation-overhang/).
+  is a situation where the value of a company just doesn’t reach the number of dollars
```

Comment:
"is a situation" and "for example" are almost never needed phrases. Would revert these changes!

Reply:
Sure re "for example" (though in general we probably should tag detailed examples).

Regarding "is a situation" the main thing is this definition was not a definition before, but a more vague statement saying it's something that "occurs". A few definitions are like this and we should clean that up.

Reply:
Rather, the previous definition indicated that it was 'something that occurs when'... i.e., a "situation," 'something that occurs'